x11-libs/libice/libice-1.0.9.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="Inter-Client Exchange Library"
+DESCRIPTION="The ICE library utilizes callbacks to process incoming \
+messages. Using callbacks allows ProtocolSetup messages and authentication \
+to happen behind the scenes. An additional benefit is that messages never \
+need to be buffered up by the library when the client blocks waiting for \
+a particular message."
+HOMEPAGE="https://www.x.org/releases/individual/lib/"
+COPYRIGHT="1993, 1998 The Open Group"
+LICENSE="MIT (no promotion)"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libICE-$portVersion.tar.bz2"
+CHECKSUM_SHA256="8f7032f2c1c64352b5423f6b48a8ebdc339cc63064af34d66a6c9aa79759e202"
+SOURCE_DIR="libICE-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
```

Comment:
```diff
-ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
```
It builds fine on both x86_gcc2 (using x86 second arch) and x86_64.
Shall we merge after marking x86_64 as tested?

Reply:
Ok, will adjust :+1: